pkg/machinery/resources/cluster/affiliate.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/talos-systems/talos/pkg/machinery/config/types/v1alpha1/machine"
 )
 
+//go:generate deep-copy -type AffiliateSpec -type IdentitySpec -type MemberSpec -type ConfigSpec -header-file ../../../../hack/boilerplate.txt -o deep_copy.generated.go .
```

Comment:
I feel this commend should go to `cluster.go` as it has nothing to do with the `Affiliate`, but rather for all types in this directory. 

I know I'm late to the party, but still ;)

And it make sense to sort types by name to make it easier to see which one is there, and which one should be added